docs/releases/5.2.txt

```diff
@@ -173,7 +173,7 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* The PostgreSQL backend no longer uses ``CASCADE`` when dropping a column.
```

Comment:
Ah, I missed that this was in the default backend, and then undone in MySQL/Oracle/SQLite. I think we probably want something more like:

```suggestion
* The base and PostgreSQL database backends no longer use ``CASCADE`` when
  dropping a column.
```

It might also be more suitable under the “Database backends“ heading.

Reply:
This is not a feature, it's a backward incompatibility change in documented behavior. Please move this to the "Backwards incompatible changes in 5.2" section.